.github/workflows/validate-new-news.yml

```
@@ -0,0 +1,39 @@
+---
+name: Validate new news entries
+
+on:
+  pull_request:  # Trigger on PRs to develop
+    branches:
+      - develop
+
+jobs:
+  run:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Get out of detached head state
+        run: |
+          git fetch origin ${{ github.base_ref }}
+          git fetch origin ${{ github.ref }}
+          git checkout FETCH_HEAD --
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Validate new news items
+        run: |
+          git diff --name-status origin/${{ github.base_ref }}
+          git diff --name-status origin/${{ github.base_ref }}\
+          | python scripts/verify-news-fragments.py
+
+      - name: Install towncrier
+        run: pip install -q -U setuptools wheel towncrier
+
+      - name: Run towncrier
+        run: towncrier --yes
```

I suggest we use `towncrier --draft` instead - it doesn't actually write anything and there's no need to write anything. The check will work just as well.
```suggestion
        run: towncrier --draft
```

---

Well it doesn't push the output so I thought it wouldn't matter.